docs/docs/cli/installation.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Installation
+sidebar_label: Installation
+hide_table_of_contents: false
+---
+
+import Highlighter from "@site/src/components/Highlighter";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Highlighter>
+Install the CLI quickly with our step-by-step guide or build it from source
+</Highlighter>
+
+## Ways to Install
```

Comment:
I think this is a strange header... and probably the document will look better without it